### PR TITLE
added new javascript question

### PR DESCRIPTION
**Pull request by Anamika2716**
added new javascript question

### CODE REVIEW

File: javascript/answers/anamika-katyalq22.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+        <div>
+            <div>
+                <h4>
+                   What is the output of the code below ?
+                </h4>
+            </div>
+            <div>
+                <strong>Answer:</strong>
+                <p>Error 1, Success4</p>
```
Comment (dakshkhetan) [inline javascript/answers/anamika-katyalq22.html:49]: @Anamika2716 please try to give an explanation for the output.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
This looks good. 
Just make the suggested change. 👍

File: javascript/answers/anamika-katyalq22.html (hunk 1 above)
Comment (Anamika2716) [reply javascript/answers/anamika-katyalq22.html:49]: Added explanation

Comment (dakshkhetan) [reply javascript/answers/anamika-katyalq22.html:49]: It does not show any explanation added. 😕 
Check if your changes were committed or not.

**Review by arpansac — CHANGES_REQUESTED** [review]
Please redo this PR completely, too many unnecessary changes

**Comment by dakshkhetan** [comment]
Closing this PR.

**Pull request closed without merging**